features/base/tracks/actions.js

```diff
@@ -97,7 +71,8 @@ export function trackAdded(track) {
                 jitsiTrack: track,
                 local: track.isLocal(),
                 mediaType: track.getType(),
-                mirrorVideo: _shouldMirror(track),
+                mirrorVideo: track.getCameraFacingMode()
+                    === CAMERA_FACING_MODE.USER,
```

Comment:
1. I'd rather we keep the _shouldMirror function because to me it clarifies the source code (apart from encapsulating the logic around the choice to mirror video or not).

2. That equality check is kind of... opportunistic. The return value of the function JitsiLocalTrack#getCameraFacingMode() is of type CAMERA_FACING_MODE but that type is defined by lib-jitsi-meet. The type CAMERA_FACING_MODE on the right side of the equality check is defined by jitsi-meet.